tests/DrushStackTest.php

```diff
@@ -97,9 +117,65 @@ public function testDrushStatus()
 
     public function testDrushVersion()
     {
-        $version = $this->taskDrushStack(__DIR__ . '/../vendor/bin/drush')
-            ->getVersion();
-        $this->assertEquals('8.1.12', $version);
+        foreach (['8.1.15' => '8.1.15', '9.0.0-rc1' => '9.0.0'] as $version => $version_string) {
+            if (version_compare('5.6', phpversion()) > 0 && version_compare($version_string, '9.0') > 0) continue;
+
+            $this->ensureDirectoryExistsAndClear($this->tmpDir);
+            $this->writeComposerJSON();
+            $this->composer('require --update-with-dependencies drush/drush:"' . $version .'" -vvv');
+            $version2 = $this->taskDrushStack($this->tmpDir . '/vendor/bin/drush')
```

Comment:
Maybe different names for the versions?

Like `composerDrushVersion`, `expectedVersion` above and `actualVersion` here?